### ISSUE TITLE
Send to multiple custom urls

### ISSUE THREAD

**Issue by PedroKTFC** [mendhak/gpslogger#1114]
I'd like it if I could use more than one custom url.

**Comment by Kolia56** [comment]
I second this request, would be very nice. Thank you

**Comment by andreapx** [comment]
@PedroKTFC @Kolia56 
I've solved this problem setting the custom URL to my Node Red instance where I receive the data and send it to 2 different URLs (Traccar and DaWarich)

**Comment by PedroKTFC** [comment]
I assume you're running NodeRed in HA. How did you target, read and process the message in NR?

**Comment by andreapx** [comment]
> I assume you're running NodeRed in HA. How did you target, read and process the message in NR?

No, I have NodeRed running on a rented public VPS (actually it runs on almost every PC/raspberry/server I have! 😂)
I've created an _HTTP in_ node with a random gibberish URL used only by GPSlogger and with 2 functions I prepare the data for Traccar and Dawrich.


![26-09-2024_19-25-43_5_Node-RED__GPS_—_Mozilla_Firefox](https://github.com/user-attachments/assets/ad0cbd1d-e4a0-43e1-b721-a59b07fce553)

The functions node are like this:
```
let newMsg = {};
newMsg.url = "http://172.20.0.45:5055/?id=" + msg.payload.id +
    "&lat=" + msg.payload.lat +
    "&lon=" + msg.payload.lon +
    "&time=" + msg.payload.time +
    "&speed=" + msg.payload.spd_kph +
    "&batt=" + msg.payload.batt +
    "&accuracy=" + msg.payload.acc +
    "&heading=" + msg.payload.dir +
    "&altitude=" + msg.payload.alt;
return newMsg;
```
